src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -119,7 +125,7 @@ public static void SameBinding_Soap11_EchoString()
     }
 
     // Client and Server bindings setup exactly the same using Soap12
-    [Fact]
+    //[Fact]
```

Comment:
Sigh :( Got it, I do the same thing, wish there was a good way to run a single test